src/test/end-to-end/pages/solicitorApplyCaveat/completeApplication/page2.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const testConfig = require('src/test/config.js');
+const completeApplicationConfig = require('./completeApplication');
+const commonConfig = require('src/test/end-to-end/pages/common/commonConfig');
+
+module.exports = async function () {
+    const I = this;
+    await I.waitForText(completeApplicationConfig.page2_waitForText, testConfig.TestTimeToWaitForText);
+
+    await I.waitForNavigationToComplete(commonConfig.goButton);
+};
```

Comment:
same here, is this step just waiting?

Reply:
this is the second page as part of complete application process. We are making sure the second page appears with the required text and make sure solicitor continues on the complete application through this page.

Reply:
and same here, the naming threw me. But not in scope. 👍